src/gluonts/dataset/split.py

```diff
@@ -81,6 +81,13 @@
 from gluonts.dataset.field_names import FieldName
 
 
+def ceildiv(a, b):
+    """
+    Ceiling division between ``a`` and ``b``.
+    """
+    return -(a // -b)
```

Comment:
What sorcery is this?

Reply:
I guess it's the same as

```py
sum(divmod(a, b))
```

Reply:
Ah, no, of course not.

Reply:
Python's `//` is the floor division, that's the usual integer division where `7 // 2 == 3` for example.

This is the ceiling division, where `7 // 2 == 4`, that is the quotient is rounded up (if it's not a whole number).